prereise/gather/demanddata/transportation_electrification/smart_charging.py

```diff
@@ -156,6 +254,34 @@ def smart_charging(
         g2v_load = np.zeros((100, 72))
         individual_g2v_load = np.zeros((nd_len, 72))
 
+        # code developer debugging variables
+        individual_vmt = 0
+        individual_trip_miles = 0
+        linprog_charge_results = 0
+        optimization_fail = 0
+        flag_1_fail = 0
+        flag_2_fail = 0
+        flag_3_fail = 0
+        flag_4_fail = 0
+        missed_vmt = 0
+        # if veh_type.lower() in {"ldv", "ldt"}:
+        #     missed_vehicles = pd.DataFrame(
+        #         columns=const.nhts_census_column_names
+        #     ).astype("int64")
+        # elif veh_type.lower() in {"mdv", "hdv"}:
+        #     missed_vehicles = pd.DataFrame(
+        #         columns=list(newdata.columns)
+        #         + [
+        #             "power_allowed",
+        #             "charging consumption",
+        #             "seg",
+        #             "power",
+        #             "segcum",
+        #             "energy limit",
+        #             "rates",
+        #         ]
+        #     )
```

Comment:
Do we need to keep these comments?

Reply:
Yes.  Ultimately, we want to uncomment these to include as a debugging option for the user / developer.  Unfortunately, at the moment, we ran into an error with `pd.concat` on line 449 that does not impact the profile generation but does impact this bonus feature.